### ISSUE TITLE
[TextInputEditText] The left padding is 12dp instead of 16dp as per material.io specs

### ISSUE THREAD

**Issue by KdB13** [material-components/material-components-android#1706]
**Description:** The implementation of TextInputEditText has different `android:paddingLeft` value than the one provided in the specifications of **Text fields**.
The recommended specs say that padding should be of `16dp` while the current implementation has only `12dp`.

**Expected behavior:** The padding should be like the following image:

![Text field, left padding](https://i.ibb.co/sgyj5z2/textfield-padding.png)

**Source code:** The style `Base.Widget.MaterialComponents.TextInputEditText` has set wrong padding of `12dp`.

    
    <style name="Base.Widget.MaterialComponents.TextInputEditText" parent="Widget.Design.TextInputEditText">
        . . .
        <item name="android:paddingStart" tools:ignore="NewApi">12dp</item>
        <item name="android:paddingEnd" tools:ignore="NewApi">12dp</item>
        <item name="android:paddingLeft">12dp</item>
        <item name="android:paddingRight">12dp</item>
        . . .
    </style>

Link to full file: https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/textfield/res/values/styles.xml

**Android API version:** Any API which supports Material Components

**Material Library version:** The latest version

**Device:** Nexus 4 (AVD)

**Comment by KdB13** [comment]
@gabrielemariotti The issue you referenced is not relevant to the current issue. They are two separate components.

**Comment by gabrielemariotti** [comment]
@KdB13 Yes sorry. I wrote the wrong issue in the PR, I changed it but the reference it still here and it seems there is no way to remove it.

**Comment by KdB13** [comment]
This commit c8197e3 fixes the issue. I am closing it. Thanks. 👍
